airflow-core/src/airflow/ui/src/i18n/locales/he/common.json

```diff
@@ -0,0 +1,79 @@
+{
+  "defaultToGraphView": "תצוגת גרף כברירת מחדל",
+  "defaultToGridView": "תצוגת רשת כברירת מחדל",
+  "logout": "התנתק",
+  "switchToDarkMode": "החלף למצב כהה",
+  "switchToLightMode": "החלף למצב בהיר",
+  "timezone": "איזור זמן",
+  "user": "משתמש",
+  "language": {
+    "chinese": "סינית מסורתית",
+    "english": "אנגלית",
+    "german": "גרמנית",
+    "korean": "קוריאנית",
+    "select": "בחר"
+  },
+  "nav": {
+    "home": "מסך הבית",
+    "assets": "נכסים (Assets)",
+    "browse": "עיין",
+    "admin": "מנהל",
+    "docs": "תיעוד",
+    "plugins": "פלאגינים"
+  },
+  "browse": {
+    "auditLog": "לוג מערכת",
+    "xcoms": "(XComs)"
+  },
+  "admin": {
+    "Variables": "משתנים",
+    "Pools": "בריכת משאבים",
```

Comment:
It's a nice translation, but it reminds me (and Google) more of a [swimming pool](https://www.google.com/search?client=firefox-b-d&channel=entpr&q=%D7%91%D7%A8%D7%99%D7%9B%D7%AA+%D7%9E%D7%A9%D7%90%D7%91%D7%99%D7%9D).
I think that we should be brave here and try to adopt the term [איגום משאבים](https://www.milononline.net/do_search.php?Q=%E0%E9%E2%E5%ED), which is not as common - but looks more appropriate for this use case.